package.json

```diff
@@ -123,6 +123,7 @@
     "intl-pluralrules": "1.1.2",
     "js-cookie": "2.2.1",
     "json2csv": "4.5.4",
+    "jsonwebtoken": "8.5.1",
```

Comment:
@whitmanschorn This was marked as resolved but I didn't get an answer to this question. It looks like `jsonwebtoken` is a dependency of `express-jwt` and should already be installed because of that package, is there a reason we need to explicitly specify a version of `jsonwebtoken` here?